internal/e2e/rollup_code_splitting/additional_entry.spec.js

```diff
@@ -8,38 +8,46 @@ describe('bundling additional entry point', () => {
     check(path, 'bundle.min.js', 'bundle-min_golden.js_');
   });
 
-  it('bundle.cs.es6 should work', () => {
+  // Disabled because native ESModules can't be loaded in current nodejs
```

Comment:
TODO for a future version of nodejs?

Reply:
filed https://github.com/bazelbuild/rules_nodejs/issues/593